store/postgres/src/chain_store.rs

```diff
@@ -364,6 +363,14 @@ mod data {
             }
         }
 
+        /// Returns a fully qualified table name to the blocks table
+        pub(super) fn blocks_table(&self) -> String {
+            match self {
+                Storage::Shared => String::from("public.ethereum_blocks"),
+                Storage::Private(Schema { name, .. }) => format!("{}.blocks", name),
+            }
+        }
+
```

Comment:
This is the proposed change to fix the improper access to a `public.blocks` table.